src/ios/view.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * test view
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { UIScreen } from 'UIKit/UIScreen';
+import { UIColor } from 'UIKit/UIColor';
+import { UIView } from 'UIKit/UIView';
+
+export default class View {
+	constructor (opts = {}) {
```

Comment:
i think our current lint setup for appc forces this but maybe not for ES6. i think it's cleaner.